docs/configuration/settings.rst

```diff
@@ -0,0 +1,586 @@
+Check the behaviour of the server with the ``url`` value returned in :ref:`the
+hello view <batch>`.
+
++---------------------------------------+--------------------------------------------------------------------------+
+| Setting name                          | What does it do?                                                         |
++=======================================+==========================================================================+
+| cliquet.http_host ``None``            | The HTTP Host used by Kinto to refer to itself. If set to `None`, the    |
+|                                       | HTTP host is read from HTTP headers.                                     |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.http_scheme ``None``          | The HTTP scheme used by Kinto to refer to itself. If set to `None`, the  |
+|                                       | HTTP scheme is read from the HTTP headers.                               |
++---------------------------------------+--------------------------------------------------------------------------+
+
+Logging
+=======
+
++---------------------------------------+--------------------------------------------------------------------------+
+| Setting name                          | What does it do?                                                         |
++=======================================+==========================================================================+
+| cliquet.logging_renderer              | The Python *dotted* location of the renderer class that should be used   |
+| ``cliquet.logs.ClassicLogRenderer``   | to render the logs to the standard output.                               |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.newrelic_config ``None``      | `None`                                                                   |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.newrelic_env ``dev``          | `dev`                                                                    |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.statsd_prefix ``cliquet``     | The prefix to use when sending data to statsd.                           |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.statsd_url: ``None``          | The URL to use to connect to the statsd host. e.g.                       |
+|                                       | ``udp://localhost:8125``                                                 |
++---------------------------------------+--------------------------------------------------------------------------+
+
+Logging with Heka
+:::::::::::::::::
+
```

Comment:
A quick shout-out to Heka might be nice here. :smile: (It could even be lifted right from their docs.)
Consider:
> Heka is an open source stream processing software system developed by Mozilla. Heka is a "Swiss Army Knife" type tool for data processing, and is useful for a wide variety of different tasks. For more information, see https://hekad.readthedocs.org/